src/test/java/com/hocs/test/glue/LoginStepDefs.java

```diff
@@ -170,8 +170,10 @@ private void navigateToHocs() {
                 case "QA":
                     baseUrl = Environments.QA.getEnvironmentURL();
                     break;
+                case "DEMO" :
+                    baseUrl = Environments.DEMO.getEnvironmentURL();
                 default:
-                    fail("Environment must be set to LOCAL, DEV or QA");
+                    fail("Environment must be set to LOCAL, DEV, DEMO or QA");
```

Comment:
```suggestion
                    fail(" baseUrl + " is not defined within " + getMethodName());
```

Without passing in `baseUrl` here, it's hard to tell what actually went wrong here. By including it in the failure condition, you can quickly see what it was at the time of failure. `getMethodName` is optional but we find it helps to include it so we can quickly find the offending method.